package Logical;

import exceptionHandling.Protected;

public class TestProtected extends Protected {

	public static void main(String[] args) {
		TestProtected t =new TestProtected();
		System.out.println(t.x);

	}

}
